Sources/SwiftDocC/SwiftDocC.docc/Resources/ThemeSettings.spec.json

```diff
@@ -856,6 +856,9 @@
                                     },
                                     "onThisPageNavigator": {
                                         "$ref": "#/components/schemas/FeatureFlagDisable"
+                                    },
+                                    "i18n": {
```

Comment:
Should we consider making this just `localization`? None of the other feature flags use abbreviations so this is standing out to me a bit.

Reply:
Ah right – maybe another reason to spell it out then since I was clearly confused. 😃

Reply:
It seems that i18n is a front-end term

- https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/i18n
- https://kazupon.github.io/vue-i18n/

I think it is fine for people working with web.

Reply:
I agree with @Kyle-Ye , we usually refer to internationalization as i18n in most of the frontend libraries.